tests/functional/conftest.py

```diff
@@ -154,8 +154,13 @@ def check_is_alive():
     Return a healthcheck function fixture for the GitLab container spinup.
     """
 
-    def _check(container):
-        logging.info("Checking if GitLab container is up...")
+    def _check(container: str, start_time: float) -> bool:
+        setup_time = time.perf_counter() - start_time
+        minutes, seconds = int(setup_time / 60), int(setup_time % 60)
+        logging.info(
+            f"Checking if GitLab container is up. "
+            f"Have been checking for {minutes} minute(s), {seconds} seconds ..."
+        )
```

Comment:
IMO for these we have the logging timestamps. Our functional test `conftest.py` is getting quite bloated just for the container startup compared to things like https://github.com/gitlabhq/terraform-provider-gitlab/blob/main/scripts/await-healthy.sh.

Reply:
True, we do have timestamps. But then it means I need to do math in my head. I hate doing math in my head 😉